swagger/flipt.swagger.json

```diff
@@ -261,7 +261,6 @@
                 }
               },
               "required": [
-                "flagKey",
```

Comment:
Was this a bug in the swagger output that has been addressed?

Seems there are a lot of required constraints being dropped by this upgrade.
But I lack the context to know if that was actually desired.

Reply:
I think its a change with grpc-gateway/swagger plugin. I think that because these fields are in the path that they are required by default, so there's no need to make them required as part of the body